python/cuml/common/handle.pyx

```diff
@@ -105,3 +107,10 @@ cdef class Handle:
     def getNumInternalStreams(self):
         cdef cumlHandle* h_ = <cumlHandle*>self.h
         return h_.getNumInternalStreams()
+
+    def __getstate__(self):
+        return self.n_streams
```

Comment:
Should RMM status be preserved?

Reply:
Given that we're making RMM enabled by default (always?), is this still needed?

Reply:
Seems weird not to preserve it either way... if we make rmm default and somebody disables it, they'd presumably be confused to see the unpickled handle flip that choice back